packages/react-instantsearch/src/core/createInstantSearch.test.js

```diff
@@ -1,100 +1,61 @@
 /* eslint-env jest, jasmine */
 /* eslint-disable no-console */
 import React from 'react';
-import {mount} from 'enzyme';
-
-import algoliaClient from 'algoliasearch';
+import {shallow} from 'enzyme';
```

Comment:
My reasoning is that if this is a unit test then reducing the unit and targeting it precisely is what I want to do. What do you think?